Source/Internal/Table/PDFTableObject.swift

```diff
@@ -395,24 +400,24 @@ class PDFTableObject: PDFRenderObject {
                                                     minOffset: minOffset,
                                                     maxOffset: maxOffset)
                 result += try sliceObject.calculate(generator: generator, container: container)
-
-                if nextPageCells.isEmpty && idx == cells.count - 1 {
```

Comment:
Unfortunately I didn't add comments 4 years ago when I added this conditional, maybe the logic has something to do with the multi-page breaking.

As the current implementation is not working, we'll go with this approach, and this comment is only for future reference